src/map/zone.h

```diff
@@ -232,7 +232,6 @@ enum ZONEID : uint16
     ZONE_DYNAMIS_BASTOK                 = 186,
     ZONE_DYNAMIS_WINDURST               = 187,
     ZONE_DYNAMIS_JEUNO                  = 188,
-    ZONE_189                            = 189, // Southern San d'Oria [S] Residential Area
```

Comment:
we actually want to keep the placeholder, and just kill the false statement about what it is.

Reply:
else we potentially have a nullptr in other places, even tho we aren't actively using that zone

Reply:
Yessir 👍